pom.xml

```diff
@@ -176,4 +220,147 @@
             </url>
         </pluginRepository>
     </pluginRepositories>
+
+    <profiles>
+        <profile>
+            <id>distribution</id>
+            <properties>
+                <jreleaser.signing.mode>MEMORY</jreleaser.signing.mode>
+                <release.jfrog.active>NEVER</release.jfrog.active>
+                <release.mavencentral.active>ALWAYS</release.mavencentral.active>
```

Comment:
Correct?

Reply:
Yes. Default is to release on Maven central, but using the properties it is possible to deploy on jfrog for test